lesson-1/.gitignore

```diff
@@ -1 +0,0 @@
-!node_modules
```

Comment:
We need to keep an ignore on `node_modules` for each lesson actually. That's because I have commited a symlink that links back to the main "shared" `node_modules` at the top level of the project. This way students only have to `yarn` once. Let's add these back but keep the top level change to ignore `local.*` files.